README.md

```diff
@@ -591,7 +636,19 @@ If you are using autodiscovery, keep a few points in mind:
 - Components inside the auto-imported files still need to be registered with `@component.register()`
 - Auto-imported component files must be valid Python modules, they must use suffix `.py`, and module name should follow [PEP-8](https://peps.python.org/pep-0008/#package-and-module-names).
 
-Autodiscovery can be disabled via in the [settings](#disable-autodiscovery).
+Autodiscovery can be disabled in the [settings](#disable-autodiscovery).
+
+### Manually trigger autodiscovery
+
+Autodiscovery can be also triggered manually as a function call. This is useful if you want to run autodiscovery at a custom point of the lifecycle:
+
+```py
+from django_components import autodiscover
+
+autodiscover()
+```
+
+Note, though, that if a module was previously imported, it's body will NOT be executed upon calling `autodiscover`.
```

Comment:
This line is needlessly complicating things IMHO. Can we remove it?

Reply:
Updated!